### ISSUE TITLE
How to add "Import HTML" Button at the top of editor?

### ISSUE THREAD

**Issue by ArchanaSharma95** [unlayer/react-email-editor#131]
I want to add "Import HTML" button on the top of page. How to add import html functionality on click of button?

**Comment by lidrissi** [comment]
you have a premium plan ?

**Comment by ArchanaSharma95** [comment]
No, but I am checking out all it's functionality and surely upgrade if it will fullfill all my requirement.

**Comment by lidrissi** [comment]
I dont think its possible here is the rest api https://docs.unlayer.com/reference#reference-getting-started

**Comment by ArchanaSharma95** [comment]
But it's response also having "design" field which return dat in json format how can I can load dat using html and it should be available with the drag and drop.
